src/validations/html/htmllint.js

```diff
@@ -46,7 +46,17 @@ const errorMap = {
 
   E008: () => ({reason: 'doctype'}),
 
-  E009: () => ({reason: 'href-style'}),
+  E009: (error, source) => {
+    const lines = source.split('\n');
+    const hrefValExpr = /href="([^"]*)/u;
+    const hrefVal = hrefValExpr.exec(
+      lines[error.line - 1].slice(error.column),
+    )[1];
+    if (hrefVal.indexOf('mailto:') > -1 || hrefVal.indexOf('tel:') > -1) {
```

Comment:
Nit, but [`startsWith()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/startsWith) is a more expressive way to do these checks